src/components/post-comment.jsx

```diff
@@ -72,6 +72,9 @@ export default class PostComment extends React.Component {
     if (shouldClearText) {
       this.setState({editText: ''});
     }
+    if (this.props.editText !== newProps.editText) {
```

Comment:
Thanks for the explanation. It still a bit unclear to me, but it seems you're confident in it, so I'm fine with it.